### PR TITLE
Scheme

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

File: types/symbol.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package types
+
+type Symbol string
+
+const (
+	Btc Network = "btc"
```
Comment (hihiboss) [inline types/symbol.go]:
```suggestion
	Btc Symbol = "btc"
```

**Comment by baumstern** [comment]
#### Suggestion for `Show Wallets`

If the main functionality is to provide identifiable values of each wallet, we could remove the `balance` field from response. 
Also, it is the only stateful value; the balance of each address changes over time. There is also performance consideration, it might take some delays to get the balance of each address from external blockchain networks.

The response of `Show Wallets` request without the `balance` field:
```json
 {
     "id": "e372988fb3b2108",
     "scheme": "multiSig",
     "addresses": [
         {
             "address": "0x3a065000ab4183c6bf581dc1e55a605455fc6d61",
             "coin": {
                 "id": "988f08b3b21e372",
                 "blockchain": {
                     "platform": "ethereum",
                     "network": "ropsten"
                 },
                 "symbol": "eth",
                 "decimals": 18,
                 "meta": {}
             }
         },
         {
             "address": "5fc6d3a065000ab418dc1e55a605453c6bf58161",
             "coin":{
                 "id": "988f08b3b21e372",
                 "blockchain": {
                     "network": "bitcoin",
                     "platform": "mainnet"
                 },
                 "symbol": "btc",
                 "decimals": 8,
                 "meta": {}
             }
         }
     ]
 }
 ```

**Comment by junbeomlee** [comment]
I agree with the performance issue when looking up the balance. However, from the user's point of view, i think they would like to check the overall balance at once, and since we do not currently support many coins, the performance issue seems to be fine now, but it seems that there will be enough problems with balance later. what do you think?

File: types/symbol.go (hunk 1 above)
Comment (junbeomlee) [reply types/symbol.go]: i fixed it

**Comment by junbeomlee** [comment]
#15

**Pull request closed without merging**